partner_compassion/models/partner_compassion.py

```diff
@@ -280,6 +315,8 @@ def write(self, vals):
         email = vals.get("email")
         if email:
             vals["email"] = email.strip()
+        if vals.get("criminal_record"):
+            vals["criminal_record_date"] = fields.Date.today()
```

Comment:
The field `criminal_record_date` will get override every time the write method is called here (if a partner has a criminal record set). Is this really the behaviour that is expected ?

Reply:
`criminal_record` will be present in `vals` only if it is updated so it won't change each time we write other fields in the partner. This is what we want in order to consider that each time we update it, it means we put a more recent document.